config/application.rb

```diff
@@ -52,16 +52,16 @@ class Application < Rails::Application
 
     # Use standalone BigBlueButton server.
     config.bigbluebutton_endpoint = if ENV["BIGBLUEBUTTON_ENDPOINT"].present?
-       ENV["BIGBLUEBUTTON_ENDPOINT"]
```

Comment:
Please exclude this file from the PR

Reply:
thx for the review - unfortunatly, I'm very busy, atm.
The line is there to make rubocop happy, by fixing identation. i'll take care of it by mid of July.

Reply:
No worries. I'll just merge it in and fix it myself. Thanks!